koku/masu/util/ocp/ocp_post_processor.py

```diff
@@ -163,7 +164,39 @@ def _generate_daily_data(self, data_frame):
 
         return daily_data_frame
 
-    def process_dataframe(self, data_frame):
+    def _remove_anomalies(self, data_frame: pd.DataFrame, filename: str) -> pd.DataFrame:
+        """Removes rows with anomalous values from the DataFrame."""
+
+        threshold_map = {
+            1e15: [
```

Comment:
I am still a little confused by how you are determining the threshold limits here. On the [Postgresql side](https://github.com/project-koku/koku/blob/main/koku/reporting/provider/ocp/models.py#L150-L153) of things everything appears to be a `DECIMAL(33, 15)`. From my understanding the maximum number that can fit into that decimal restriction is `1e18`.

Granted when we [build the managed table](https://github.com/project-koku/koku/blob/main/koku/masu/database/trino_sql/reporting_ocpusagelineitem_daily_summary.sql#L275-L288) in Trino we transform the seconds into hours.

So wouldn't our threshold for the maximum number of seconds we can accept here be:
```1e18 * 3600```